pkg/apiserver/profiling/model.go

```diff
@@ -23,13 +23,14 @@ const (
 )
 
 type TaskModel struct {
-	ID          uint                    `json:"id" gorm:"primary_key"`
-	TaskGroupID uint                    `json:"task_group_id" gorm:"index"`
-	State       TaskState               `json:"state" gorm:"index"`
-	Target      model.RequestTargetNode `json:"target" gorm:"embedded;embedded_prefix:target_"`
-	FilePath    string                  `json:"file_path" gorm:"type:text"`
-	Error       string                  `json:"error" gorm:"type:text"`
-	StartedAt   int64                   `json:"started_at"` // The start running time, reset when retry. Used to estimate approximate profiling progress.
+	ID                uint                    `json:"id" gorm:"primary_key"`
+	TaskGroupID       uint                    `json:"task_group_id" gorm:"index"`
+	State             TaskState               `json:"state" gorm:"index"`
+	Target            model.RequestTargetNode `json:"target" gorm:"embedded;embedded_prefix:target_"`
+	FilePath          string                  `json:"file_path" gorm:"type:text"`
+	Error             string                  `json:"error" gorm:"type:text"`
+	StartedAt         int64                   `json:"started_at"` // The start running time, reset when retry. Used to estimate approximate profiling progress.
+	ProfileOutputType string                  `json:"profile_output_type"`
```

Comment:
Set the ProfileOutputType as string since we will have other output types is the future. It's more flexible to use string than use bool.